src/main/java/io/wisetime/idea/branch/BranchHelper.java

```diff
@@ -32,10 +30,7 @@ public void onBranchChanged(String branchName) {
 
   private void updateFrameTitle() {
     final String projectTitle = FrameTitleBuilder.getInstance().getProjectTitle(project);
-    SettingsState settings = SettingsState.getInstance();
-    if (settings.appendToProjectName) {
-      updateProjectTitle(projectTitle);
-    }
+    updateProjectTitle(projectTitle);
```

Comment:
we should also consider wrapping this so that if there is an issue updating the branch window, it does not throw an exception to user and rather logs an error